src/cljs/org/broadinstitute/firecloud_ui/page/workspace/method_config_importer.cljs

```diff
@@ -119,9 +150,11 @@
                      :padding "20px 48px 18px"}}
        [:div {:style {:fontSize 24 :align "center" :textAlign "center" :paddingBottom "0.5em"}}
         "Select Method Configurations For Import"]
-       [ImportWorkspaceMethodsConfigurationsList {:method-confs (utils/parse-json-string
-                                                                  (utils/->json-string
-                                                                    (create-mock-methodconfs-import)))}]
+         [
```

Comment:
Are you referring to the outer-most "[" and "]" here containing ONLY an if-statement and its true and false blocks and an "x" placeholder?
If yes, then it seems they are needed.  I tried with and without the brackets and without them the import-overlay doesn't use the render-import-overlay-react created in the "let" about 20 lines above.  Maybe you want a refactor with a larger scope (than the brackets)?